samples/zigbee/light_switch/src/main.c

```diff
@@ -434,17 +446,74 @@ void zboss_signal_handler(zb_bufid_t bufid)
 	}
 }
 
-void main(void)
+#if CONFIG_NUS_CMD
+
+static void turn_on_cmd(struct k_work *item)
+{
+	ARG_UNUSED(item);
+	zb_buf_get_out_delayed_ext(light_switch_send_on_off,
+				   ZB_ZCL_CMD_ON_OFF_ON_ID, 0);
```

Comment:
As far as I remember, this command is not handled by platform code. If you wish to use it - please extend the implementation, so NUS commands will be able to wake-up ZBOSS thread.

Alternatively, generate a user event in every command handler.